signal-hook-mio/src/lib.rs

```diff
@@ -0,0 +1,145 @@
+//! A module for integrating signal handling with the MIO runtime.
```

Comment:
I will add this to the crates.

Is there a reason why it is `https://docs.rs/signal-hook/0.1.16/signal-hook/` in the file you linked? The ending in `signal-hook` seems wrong to me because this makes the link target not exist.

The [example for html_root_url](https://doc.rust-lang.org/rustdoc/the-doc-attribute.html#html_root_url) also stops after the version.

If this is an error I can fix it as well.

Reply:
Yes, this is a mistake, the part there will redirect to `signal_hook` (with underscore), as that's the package name not crate name (or the vice versa) ‒ basically, rust identifiers don't like dashes :-|.

I'll be happy for a fix.